docs/development/releasing.rst

```diff
@@ -115,7 +115,12 @@ packages that use the full bugfix/maintenance branch approach.)
 
    Don't forget to remove any non-committed files with::
 
-      $  git clean -dfx
+      $ git clean -dfx
+
+#. Make sure the source distribution doesn't inherit limited permissions
+   following you default umask::
+
+     $ umask 0022
```

Comment:
I would rather not go down into the rabbit hole of adjusting this document with future windows releasers in mind, I assume many of the other commands aren't working either as copy pastes, are they?